src/pykeen/models/unimodal/conv_kb.py

```diff
@@ -96,10 +96,12 @@ def __init__(
             preferred_device=preferred_device,
             random_seed=random_seed,
         )
+        regularizer = self._instantiate_default_regularizer()
         # In the code base only the weights of the output layer are used for regularization
         # c.f. https://github.com/daiquocnguyen/ConvKB/blob/73a22bfa672f690e217b5c18536647c7cf5667f1/model.py#L60-L66
-        self.append_weight_regularizer(
-            parameter=self.interaction.parameters(),
-            regularizer=self._instantiate_default_regularizer(),
-        )
+        if regularizer is not None:
+            self.append_weight_regularizer(
+                parameter=self.interaction.parameters(),
+                regularizer=regularizer,
+            )
```

Comment:
making mypy happy is sometimes dangerous, cf. https://github.com/pytorch/pytorch/pull/42356

Reply:
haha you were the one who wrote that? I've read that code before 😆

Reply:
But here I don't think that this is actually wrong. We do need to handle the possibility that the regularizer is None, since we're moving away from the NopRegularizer

Reply:
I didn't write the first part, only moved the method outside :smile:

Reply:
But it didn't solve all issues, cf. https://github.com/pytorch/pytorch/pull/42356#issuecomment-676865819